mobile-widgets/qml/DiveSummary.qml

```diff
@@ -110,7 +110,7 @@ Kirigami.ScrollablePage {
 						id: headerLabel
 						verticalAlignment: Qt.AlignVCenter
 						colorBackground: parent.color
-						text: header !== undefined ? header : ""
+						text: header !== undefined ? "   " + header : ""
```

Comment:
why is there the hard white space? Are you trying to create some margin on the left?
Happy to do that in a more QML "native way" :-)